### PR TITLE
Add exclude option

### LINKED ISSUE #10: Include an 'exclude' option
Similar to https://github.com/sindresorhus/camelcase-keys

```
exclude
Type: string[] RegExp[]
Default: []

Exclude keys from being snake_cased.
```

### PR DESCRIPTION
**Pull request by A-Boudi**
This resolves #10.

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -3,10 +3,16 @@
 var map = require('map-obj')
 var snakeCase = require('to-snake-case')
 
+const has = (array, key) => array.some(x => typeof x === 'string' ? x === key : x.test(key));
+
 module.exports = function (obj, options) {
   options = Object.assign({deep: true}, options)
+  const {exclude} = options;
```
[the remark this answers is not in the document]
Comment (A-Boudi) [reply index.js]: I wanted to delete exclude from options before passing it to map.

[the remark this answers is not in the document]
Comment (bendrucker) [reply index.js]: This doesn't do that, it's just `exclude = options.exclude`

**Comment by bendrucker** [comment]
Awesome, thanks!

File: index.js (hunk 1 above)
[the remark this answers is not in the document]
Comment (A-Boudi) [reply index.js]: Yes, I forgot to add `delete options.exclude` after it. I just wanted to know if we should address this now or when `map` would be implementing an exclude option.

**Pull request merged**